content/en/news/2024/stream-recap-week-9.org

```diff
@@ -36,7 +36,8 @@ Mostly got through a *lot* of story scenes after the last full moon operation an
 <!--TODO add embed after video is published-->
 #+end_export
 * Next Week
-Kicking off a three week Final Fantasy Challenge! I'm going to try and get through all the Final Fantasy games...or at least see how many I can get through until Aug!
++Kicking off a three week Final Fantasy Challenge! I'm going to try and get through all the Final Fantasy games...or at least see how many I can get through until Aug!+
```

Comment:
Yeah, this was taken from my last stream recap before the ff challenge 😅 Change you change it to:
Next week, we continue with a few more evening streams! The week will start with a special art stream! I'll make a new frame for art streams and maybe update my other frames. I'll also work on beating Hornet and Throwback Thursday will have a special game by special request! I hope to see you there on Twitch (link to my channel)

Thank you!